### ISSUE TITLE
harder pins on python environment

### ISSUE THREAD

**Issue by 12rambau** [openforis/sepal#260]
Every time a new build of SEPAL is released the Python environment slightly changes including libs that are complicated to deal with, namely: gdal, pyproj, tensorflow and scikitlearn. 

The 2 first because they rely on C-compiled libs and the 2 last because they quite often change the pickle file format making them unreadable between versions. In the current env, if you run any pandas call you'll get a numexpress deprecation warning. 

In short, every time there is a surprise. We are not running any tests on this env so we cannot see them until it's too late. for the sake of robustness, what would you think if we add harder pins on lib versions? 
Then they would be updated only when we decide and test them.

**Comment by TmtStss** [comment]
As a user, I have seen the numexpr deprecation warning. So far I have neglected it. Thanks to this thread, now I understand it. Still, it would be useful to solve this issue to limit confusion generated by such warning messages.

**Comment by 12rambau** [comment]
We are currently changing ubuntu distribution to jammy. We will work on the python versions afterward once the new SEPAL image is stable
